src/utils/removeListening.ts

```diff
@@ -0,0 +1,8 @@
+import { NICKNAME_SUFFIX } from "../constants/messagingConstants";
+
+export const removeListening = (nick: string) => {
```

Comment:
here we are passing a suffix and not a nickName during function invocation, can we change the parameter name to be more apt ? 
Suggestion : **suffix**

Reply:
nope ! here we are passing nickname only